target/linux/ath79/base-files/etc/board.d/01_leds

```diff
@@ -49,6 +49,9 @@ pcs,cr3000)
 	ucidef_set_led_switch "lan3" "LAN3" "pcs:blue:lan3" "switch0" "0x10"
 	ucidef_set_led_switch "lan4" "LAN4" "pcs:blue:lan4" "switch0" "0x02"
 	;;
+rosinson,wr818)
+	ucidef_set_led_netdev "wifi2g" "WIFI2G" "rosinson:red:wifi2g" "wlan0" "link"
```

Comment:
There is no such led in the dts.

Reply:
Since you've set default trigger for this LED in dts, this isn't needed. And I think we shouldn't use netdev for wifi when there is already a phy0tpt trigger available.